# Leave this in since it has no effect until the below issue is fixed
# https://github.com/bazel-contrib/rules_oci/issues/855
# Use podman prior to resolution
--- oci/private/load.sh.tpl
+++ oci/private/load.sh.tpl
@@ -16,2 +16,4 @@
-elif command -v podman &> /dev/null; then
-    CONTAINER_CLI="podman"
+elif command -v podman &> /dev/null; then
+    CONTAINER_CLI="podman"
+elif command -v container &> /dev/null; then
+    CONTAINER_CLI="container"
@@ -31,1 +31,1 @@
-"$CONTAINER_CLI" load --input <(
+"$CONTAINER_CLI" image load --input <(